images/openwisp_openvpn/openvpn_utils.sh

```
@@ -0,0 +1,88 @@
+	curl --silent --show-error --fail --insecure --output "$output_path" \
+	${DASHBOARD_INTERNAL}/admin/pki/ca/x509/ca/${CA_UUID}.crl
+}
+
+function crl_download {
+	curl --silent --show-error --fail --retry 10 --retry-delay 5 --retry-max-time 300 \
+		--insecure --output revoked.crl \
+		${DASHBOARD_INTERNAL}/admin/pki/ca/x509/ca/${CA_UUID}.crl
+}
+
+function crl_download_if_changed {
+	local tmp_crl="revoked.crl.new"
```

Shellcheck (0.11.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)